content/news/2025-12-15-fq2bam/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "GPU-accelerated genome mapper Parabricks FQ2BAM is now in Galaxy"
+date: "2025-12-15"
+authors: Anup Kumar 
+tease: "NVIDIA Parabricks GPU-accelerated FQ2BAM is now available in Galaxy, bringing FASTQ-to-BAM alignment runtime to just a few minutes for 2.5 GB paired-end human whole-genome datasets. Its runtime is benchmarked against BWA-MEM2 and BWA across five runs and shows that mapping and QC metrics remain essentially identical to CPU-based alignment—while delivering a major speedup for Galaxy workflows."
+subsites: [global,eu,us,freiburg]
+---
+
+# Parabricks FQ2BAM mapper
+
+NVIDIA's [Parabricks](https://docs.nvidia.com/clara/parabricks/latest/gettingstarted/gettingthesoftware.html#running-nvidia-parabricks) developed a GPU-accelerated tool suite for genomic analysis. One of the tools from the suite is [FQ2BAM](https://docs.nvidia.com/clara/parabricks/latest/documentation/tooldocs/man_fq2bam.html) which accelerates mapping of DNA reads. It is now integrated into Galaxy and runs on a GPU to generate read alignments in a few minutes for zipped paired-end FASTQ reads of size 2.5 GB. It is a drop-in replacement for BWA-MEM2 while maintaining the same mapping quality. [FQ2BAM](https://docs.nvidia.com/clara/parabricks/latest/documentation/tooldocs/man_fq2bam.html) tool wraps BWA-MEM2 to perform alignment from FASTQ to BAM and then handles downstream processing such as sorting, with optional steps for duplicate marking and base-quality score recalibration.
```

Comment:
For "same mapping quality" do you mean "mapping quality value" (which is a confidence score in the placements), or a more general "quality" of the mappings (such as the number of reads mapped, the coverage of the genomes, the percentage of valid pairs, etc)